pipewire/GNUmakefile

```diff
@@ -0,0 +1,17 @@
+MAKEFLAGS ::= -rR
+override CC ::= $(or $(CC),gcc)
+override CFLAGS ::= $(CFLAGS) $(shell pkg-config --cflags libpipewire-0.3 vchan-xen) -fno-strict-aliasing -fno-strict-overflow -fno-delete-null-pointer-checks -fPIC -Wall -Wextra -Werror -Wno-missing-field-initializers -Wno-unused-parameter -ggdb -O2 -D_FORTIFY_SOURCE=2
+override LDLIBS ::= $(LDFLAGS) $(shell pkg-config --libs libpipewire-0.3 vchan-xen) -lqubesdb -ggdb
```

Comment:
Please, don't use `override` (or other ways to override explicitly requested build options). This should be `CC ?= gcc`, and `..FLAGS += ...`.

Reply:
I dropped the use of `override` for `CFLAGS` and `LDLIBS`, but doing so for `CC` ran into a GNU make bug (`CC` got set to `cc`, not `gcc`).  Therefore, `override` is still used for `CC`, but if `CC` is set in any way other than by a GNU make default, a conditional will choose the existing value.

Reply:
It isn't GNU make bug, it's a bug in your makefile (or your assumptions). `cc` is a perfectly valid value for `CC` variable.

Reply:
`cc` _is_ a valid value for `CC`.  The bug is that I used `MAKEFLAGS ::= -rR` to tell GNU make _not_ to set any implicit variables, but it sets this variable anyway.  That said, `cc` is probably a better default than `gcc`, as there are platforms where `gcc` is _not_ the system C compiler.

Reply:
> I used `MAKEFLAGS ::= -rR`

Okay, that's another thing that shouldn't be there. I'm not sure why you hate those defaults so much (and I do _not_ want to know), but please stop.

Reply:
Oh, and also: please use _standard_ `Makefile` name, not `GNUmakefile`.